js/interface.js

```diff
@@ -536,9 +540,15 @@ function uploadFiles(files) {
   });
   $progress.removeClass('hidden');
 
+  if (currentFolderId == null && currentAppId == null) {
+    currentOrganisationId = currentOrgId;
+  } else {
+    currentOrganisationId = null;
+  }
   Fliplet.Media.Files.upload({
     folderId: currentFolderId,
     appId: currentAppId,
+    organizationId : currentOrganisationId,
```

Comment:
please check our code guidelines, there should not be a space after the property name

Reply:
Fixed.